testing_and_setup/compass/ocean/global_ocean/EC15to60/template_forward.xml

```diff
@@ -0,0 +1,12 @@
+<template>
+	<namelist>
+		<option name="config_dt">'00:10:00'</option>
+		<option name="config_btr_dt">'00:00:20'</option>
+		<option name="config_pio_num_iotasks">4</option>
+		<option name="config_pio_stride">36</option>
+		<option name="config_run_duration">'0000_01:00:00'</option>
+		<option name="config_mom_del4">1.5e10</option>
+		<option name="config_hmix_scaleWithMesh">.true.</option>
+		<option name="config_use_GM">.true.</option>
```

Comment:
Probably need to be adjusted

Reply:
@vanroekel, this would also be where to add other namelist changes.  Hit the "Insert a suggestion" icon and edit the code as you would like.